eg/proximity-hcsr04.js

```diff
@@ -0,0 +1,19 @@
+var five = require("johnny-five.js"),
```

Comment:
`"johnny-five.js"` => `"../"` (this is automatically converted by the examples task)